docs/assets/style.css

```diff
@@ -0,0 +1,13 @@
+html,
+body {
+  min-height: 100vh;
+}
+
+.footer {
+  position: sticky;
+  top: 100vh;
+}
```

Comment:
could we just use class="min-vh-100 position-sticky" in the HTML ?

Reply:
This is an optional comment. As there is a style.css file anyway, no problem to keep thes rules